source/computer/commands.rst

```diff
@@ -66,51 +66,688 @@ Linux/macOS 下有成百上千个命令，每个命令都有众多选项。这
         - `ssh`_
         - `scp`_
         - `rsync`_
+        - `wget`_
+        - `sed`_
+        - `diff`_
+        - `df`_
+        - `du`_
+        - `top`_
 
 ----
 
 cat
 ---
 
+``cat`` 命令名来源于 con\ **cat**\ enate（即“拼接”）。该命令可以输出某个文件的
+内容，或将多个文件\ **拼接**\ 起来。
+
+将一个文件的内容打印到标准输出::
+
+    $ cat file
+
+将文件 :file:`file1` 和 :file:`file2` 合并到一个文件 :file:`target_file` 中::
+
+    $ cat file1 file2 > target_file
+
+打印文件内容并显示行号::
+
+    $ cat -n file
+
+diff
+----
+
+``diff`` 命令的命名来自 **diff**\ erence。该命令可以用来逐行比较文件的异同::
+
+    $ diff file1 file2
+
+df
+--
+
+``df`` 命令的命名来自 **d**\ isk **f**\ ree（可使用的磁盘空间）。该命令可以获取硬盘被占用了多少空间，
+目前还剩下多少空间等信息。默认显示单位为 KB::
+
+    $ df
+
+以可读性较高的方式来显示信息，即使用 ``-h`` 或 ``--human-readable`` 选项::
+
+    $ df -h
+
+du
+--
+
+``du`` 命令的名字来自 **d**\ isk **u**\ sage（磁盘使用情况）。该命令可以查看磁盘的使用空间。
+但与 `df`_ 命令不同的是，\ ``du`` 命令用于查看文件和目录磁盘使用的空间。
+
+查看当前目录及其子目录所占磁盘空间::
+
+    $ du
+
+查看指定目录及其子目录所占磁盘空间::
+
+    $ du ~/Downloads
+
+查看指定文件所占磁盘空间::
+
+    $ du ~/Downloads/TauP-2.4.5.tar.gz
+
+以可读性较高的方式来显示信息，即使用 ``-h`` 或 ``--human-readable`` 选项::
+
+    $ du -h
+    $ du -h ~/Downloads
+    $ du -h ~/Downloads/TauP-2.4.5.tar.gz
+
 find
 ----
 
+``find`` 命令用来查找指定目录下的子目录和文件，并显示查找到的子目录和文件。
+
+列出 :file:`~/src` 目录及其子目录下的所有目录和文件::
+
+    $ find ~/src
+
+查找 :file:`~/src` 目录及其子目录下以 :file:`.c` 结尾的目录和文件::
+
+    $ find ~/src -name "*.c"
+
+同上，但忽略大小::
+
+    $ find ~/src -iname "*.c"
+
 gawk
 ----
 
+``awk`` 命令的命名起源于其三位作者的姓氏首字母。该命令可以选择标准输入、其它命令的输出或文件中
```

Comment:
Add this link: https://www.ruanyifeng.com/blog/2018/11/awk.html?

Reply:
done